extensions/ql-vscode/src/extension.ts

```diff
@@ -203,6 +203,7 @@ function getCommands(
       },
       {
         title: "Restarting Query Server",
+        cancellable: true,
```

Comment:
I'm not sure if making these cancellable makes sense. It doesn't look like this will actually cancel the restart, but will instead only cancel some tasks that are run after the restart. This is probably not what the user would expect.

A quick search of those handlers only turns up [`reregisterDatabases`](https://github.com/github/vscode-codeql/blob/31cfaebb598afb144b833beba85fc7ddc9d81dea/extensions/ql-vscode/src/databases/local-databases/database-manager.ts#L421), which actually doesn't use the token at all.